pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -266,76 +266,67 @@ def evaluate(self, model, dataloaders, max_batches, test=False):
 
     def run_evaluation(self, test=False):
         # when testing make sure user defined a test step
-        can_run_test_step = False
+        if not (self.is_overriden('test_step') and self.is_overriden('test_end')):
+            m = '''You called .test() without defining a test step or test_end.
```

Comment:
pls add ` around functions and variables

Reply:
Done